src/VCR/Util/StreamProcessor.php

```diff
@@ -629,9 +638,13 @@ public function detachCodeTransformer(AbstractCodeTransform $codeTransformer)
      */
     protected function appendFiltersToStream($stream)
     {
+        if (\count(static::$codeTransformers) === 0) {
+            return;
+        }
         foreach (static::$codeTransformers as $codeTransformer) {
             stream_filter_append($stream, $codeTransformer::NAME, STREAM_FILTER_READ);
         }
+        $this->isTransforming = true;
```

Comment:
I think I would implement this like this:

```php
if (!empty(static::$codeTransformers)) {
    $this->isTransforming = true;
}
```

In my opinion that would be more readable.

Reply:
Yes that would be better readable :slightly_smiling_face: